frontend/src/common/form.types.ts

```diff
@@ -24,18 +36,21 @@ export enum FormActionType {
 
 interface HandleInputDataAction {
   type: FormActionType.HANDLE_INPUT_DATA;
-  field: keyof FormState["inputData"];
+  field: keyof SignupFormState["inputData"];
```

Comment:
Should we create a union type `type FormState = SignupFormState | LoginFormState` and put it here, rather than relying only on `SignupFormState`?